src/plugins/query_enhancements/public/plugin.tsx

```diff
@@ -42,133 +42,137 @@ export class QueryEnhancementsPlugin
     { data, usageCollection }: QueryEnhancementsPluginSetupDependencies
   ): QueryEnhancementsPluginSetup {
     const { queryString } = data.query;
-    const pplSearchInterceptor = new PPLSearchInterceptor({
-      toasts: core.notifications.toasts,
-      http: core.http,
-      uiSettings: core.uiSettings,
-      startServices: core.getStartServices(),
-      usageCollector: data.search.usageCollector,
-    });
 
-    const sqlSearchInterceptor = new SQLSearchInterceptor({
-      toasts: core.notifications.toasts,
-      http: core.http,
-      uiSettings: core.uiSettings,
-      startServices: core.getStartServices(),
-      usageCollector: data.search.usageCollector,
-    });
+    // Initialize local storage keys when the plugin loads, if they don't exist
+    if (!window.localStorage.getItem('hasSeenSQLInfoBox')) {
+      window.localStorage.setItem('hasSeenSQLInfoBox', 'false');
+    }
+    if (!window.localStorage.getItem('hasSeenPPLInfoBox')) {
+      window.localStorage.setItem('hasSeenPPLInfoBox', 'false');
+    }
```

Comment:
Why do we ned this? why not just use undefined as the intiial state. If the key does not exist, its false by default

Reply:
Fixed.